app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -212,6 +212,13 @@ def finish(*_args)
 
   alias_method :abort_job, :cleanup
 
+  def queue_callback(state, msg, result)
```

Comment:
- I am not sure about the first point because this function is a general callback and can be used for different purposes in the future..
- You are right, I will see if I can see the current job state not already being abort before I call abort.